<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerEditor</class>
 <widget class="QMainWindow" name="TriggerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigger Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/editor.qrc">
    <normaloff>:/ico101.ico</normaloff>:/ico101.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="triggerView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="triggerNameBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Category:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="triggerPathBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Trigger Functions:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="functionView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>2</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="rootIsDecorated">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>1</string>
              </property>
             </column>
             <item>
              <property name="text">
               <string>(No Trigger Selected)</string>
              </property>
              <property name="icon">
               <iconset resource="../qrc/editor.qrc">
                <normaloff>:/ico101.ico</normaloff>:/ico101.ico</iconset>
              </property>
              <item>
               <property name="text">
                <string>Events</string>
               </property>
               <property name="icon">
                <iconset resource="../qrc/editor.qrc">
                 <normaloff>:/Trigger.png</normaloff>:/Trigger.png</iconset>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Conditions</string>
               </property>
               <property name="icon">
                <iconset resource="../qrc/editor.qrc">
                 <normaloff>:/ActiveQuestIcon.png</normaloff>:/ActiveQuestIcon.png</iconset>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Actions</string>
               </property>
               <property name="icon">
                <iconset resource="../qrc/editor.qrc">
                 <normaloff>:/Trade_Engineering.png</normaloff>:/Trade_Engineering.png</iconset>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
               </property>
              </item>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
   </widget>
   <addaction name="menu_Project"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionNewCategory"/>
   <addaction name="actionNewTrigger"/>
   <addaction name="separator"/>
   <addaction name="actionAddEvent"/>
   <addaction name="actionAddAction"/>
   <addaction name="actionRemove"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Save.png</normaloff>:/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionObjectEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Units.png</normaloff>:/Units.png</iconset>
   </property>
   <property name="text">
    <string>Object Editor</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionTriggerEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Trigger.png</normaloff>:/Trigger.png</iconset>
   </property>
   <property name="text">
    <string>Trigger Editor</string>
   </property>
   <property name="toolTip">
    <string>Trigger Editor</string>
   </property>
  </action>
  <action name="actionNewCategory">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Folder.png</normaloff>:/Folder.png</iconset>
   </property>
   <property name="text">
    <string>New Category</string>
   </property>
   <property name="toolTip">
    <string>New Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionNewTrigger">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/File.png</normaloff>:/File.png</iconset>
   </property>
   <property name="text">
    <string>Add Trigger</string>
   </property>
   <property name="toolTip">
    <string>Add Trigger</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAddEvent">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Trigger.png</normaloff>:/Trigger.png</iconset>
   </property>
   <property name="text">
    <string>Add Event</string>
   </property>
  </action>
  <action name="actionAddAction">
   <property name="icon">
    <iconset resource="../qrc/editor.qrc">
     <normaloff>:/Trade_Engineering.png</normaloff>:/Trade_Engineering.png</iconset>
   </property>
   <property name="text">
    <string>Add Action</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/editor.qrc"/>
 </resources>
 <connections/>
</ui>
